app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,64 @@
+class RentalsController < ApplicationController
+  SORT_CATEGORIES = ["movie_id", "title", "customer_id", "name", "postal_code", "checkout_date", "due_date"]
+
+  def create
+    if params[:movie_id].nil? || params[:customer_id].nil?
+      render json: { ok: false, "errors" => ["unable to create rental"]}, status: :bad_request
+      return
+    end
+    if Movie.find(params[:movie_id]) && Customer.find(params[:customer_id])    
+      rental = Rental.new(movie_id: params[:movie_id], customer_id: params[:customer_id])
+      rental.checkout_date = Date.today
+      rental.due_date = Date.today + 7 
+      
+      if rental.movie.available_inventory > 0 && rental.save
```

Comment:
Determining what is/isn't business logic is definitely a slowly acquired taste and y'all did a great job.
I wanted to add that the logic to determine whether something is safe to execute (in this case whether the rental is safe to save) is safest in its own model method so it's very clear where/how that judgement call is being made. 
This is definitely a level beyond what we've taught so far but wanted to throw it out there.